package.json

```diff
@@ -5,17 +5,23 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
+    "clean": "rm -rf ./dist && rm -rf ./node_modules",
+    "dev": "./node_modules/.bin/nodemon ./src/index.ts",
     "start": "node ./dist/index.js",
     "test": "vitest run",
-    "lint": "rome check src"
+    "lint": "rome check src",
+    "preinstall": "if [ ! -f \"/is-docker\" ]; then echo \"This script must be run inside our Docker container\" >&2; npm run clean; exit 1; fi"
```

Comment:
only allows install if in docker container. We have to remove node_modules because, somehow, it will actually do the install first?